docs/converting-to-kotlin.md

```diff
@@ -0,0 +1,7 @@
+# Converting to Kotlin
+
+The conversion of our Java code to Kotlin is still in progress. To facilitate this process, please
```

Comment:
1. ➕ to @AjeshRPai suggestions above! ❤️ 
2. Additional suggestions:
    1. Reword `the conversion the class` to `that matter`.
    2. After applying all suggestions, consider throwing all that to [ChatGPT](https://chat.openai.com/) and ask it for proofreading, we are non-native speakers and I find this helping me a lot lately.
 
PS: This is what I got out of [ChatGPT](https://chat.openai.com/):

```
# Converting to Kotlin

The conversion of Java code to Kotlin is in progress. To contribute, please follow these tips:
- If you are making changes to a Java class for your feature/fix PR, consider migrating that class to Kotlin in the same PR as a separate commit.
- If the class is too large for converting in the feature/fix PR, you can open a separate PR for that matter. This will help prevent breaking the focus of the feature/fix PR.
- Consider adding or updating unit tests. They can help avoid accidentally breaking anything while migrating to Kotlin.
```

😊

Reply:
❤️ Thanks! Updated: https://github.com/wordpress-mobile/WordPress-Android/commit/4b35b4519e9386c4f9f687ab2fb42d4bff6867f3